### DISCUSSION TITLE
VSCode no longer reopens .pdf files where I left off

### DISCUSSION THREAD

**Discussion by aryayae** [James-Yu/LaTeX-Workshop#4332] [Q&A]
For the last couple years I've been using Latex-Workshop not only as a Latex editor, but also as my primary PDF viewer.  I liked that I can have different workspaces for different subjects/projects, and whenever I reopen a workspace it not only remembers all the PDFs I had open, but it also remembers the exact pages I was on for each one (extremely useful when referencing dozens of sources for research).  I'm not sure what happened, but sometime around June it stopped functioning that way.  Now, when I close VSCode and reopen the .workspace file all the PDFs I had open get reset to the first page.

I tried to find a setting that might control this, but I couldn't find it.  I also can't find any other mention of this problem on the internet.  Does anyone know what changed or how to revert it to its previous behavior?

**Comment by riasat-sheikh** [comment]
By any chance, are you also cleaning the `*.synctex` file after compilation?
